src/models/currency.model.ts

```diff
@@ -11,7 +10,4 @@ export class Currency {
   @Column({length: 50, name: 'name'})
   @ApiProperty({required: true, type: 'string'})
   name: string;
-
-  @OneToMany(() => TransactionCard, (transactionCard) => transactionCard.currency)
-  transactionCards: TransactionCard[];
```

Comment:
Your code is right but I removed it because no need to get all cards with a specific currency.
@RamaMahjoub

Reply:
👍🏻